src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(a) {
+    return function(b) {
+      count++;
+
+      if (count <= 3 || count % 2 !== 0) {
+        const sum = a + b;
+
+        return sum;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
[CODE STYLE] - DRY. When you do similar logic, try to combine conditions. Here, you can simplify the conditional logic by checking for count > 3 and count % 2 === 0 as a single case to return 'Bzzz... Error!' and return the sum in all other cases.